packages/vehicle-manufacture/installers/hlfv1/build.sh

```diff
@@ -18,23 +18,29 @@ ROOT=$DIR/../..
 cd $ROOT
 npm install
 
+NETWORK_VERSION=$(grep -o '"version": *"[^"]*"' $ROOT/node_modules/vehicle-manufacture-network/package.json | grep -o '[0-9]\.[0-9]\.[0-9]')
```

Comment:
Impressive use of `grep`... would have used sed or jq myself... but it works :-)